README.md

```diff
@@ -31,6 +31,11 @@
   * "Walking Away From Your Open Source Project: John Resig" ([interview](https://www.youtube.com/watch?v=K9HGec2RA-Q))
 * [@jessfraz](https://github.com/jessfraz), [Kubernetes](https://github.com/kubernetes)
   * "Maintainer Stories: Jess Frazelle ([interview](https://github.com/open-source/stories/jessfraz))
+* [@kentcdodds](https://github.com/kentcdodds), [various](https://github.com/kentcdodds)
+  * "The First Pull Request" ([video](https://www.youtube.com/watch?v=HjgZQeMrw6c&list=PLV5CVI1eNcJgNqzNwcs4UKrlJdhfDjshf), [slides](http://slides.com/kentcdodds/1st-pr))
```

Comment:
Think this one is more about contributing than maintaining. Could you remove in favor of the others?

Reply:
The first half is about contributing, the second half is about how to maintain a project in a way that's conducive to first timers.

Reply:
Happy to remove it if that's still not what you're looking for though.

Reply:
ah didn't see the 2nd part! that's perfect, thanks 👍